learn/yaml-ld.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: default
+title: YAML-LD
+---
+
+<div class="container">
+  <br>
+  <div id="publ"  class="span12 text-justify">
+    <h1>YAML-LD</h1>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="span3">
+      <h2><span class="icon-pencil"> </span>YAML</h2>
+      <p class="text-justify">
+        <a href="https://yaml.org">YAML</a>
+        (Yet Another Markup Language) is a language designed for
+        minimalistic and concise data representation. It is widely used for
+        configuration files and metadata, and is optimized to be written
+        and read by humans. Hence its lack of braces and support of comments.
+      </p>
+    </div>
+    <!-- End span -->
+
+    <div class="span6">
+      <h2 style="text-align: center;">A Simple Example</h2>
+      <pre>
+<span style="color: rgb(255, 122, 0);">"@context"</span>: <a href="https://json-ld.org/contexts/person.jsonld">https://json-ld.org/contexts/person.jsonld</a>
+<span style="color: rgb(255, 122, 0);">"@id"</span>: <a href="http://dbpedia.org/resource/John_Lennon">http://dbpedia.org/resource/John_Lennon</a>
+<span style="color: rgb(255, 122, 0);">name</span>: <span style="color: rgb(0, 148, 0);">John Lennon</span>
+<span style="color: rgb(255, 122, 0);">born</span>: <span style="color: rgb(0, 0, 175);">1940-10-09</span>
+<span style="color: rgb(255, 122, 0);">spouse</span>: <a href="http://dbpedia.org/resource/Cynthia_Lennon">http://dbpedia.org/resource/Cynthia_Lennon</a></pre>
+    </div>
+    <!-- End span -->
+
+    <div class="span3">
+      <h2>
+        <span>YAML-LD
+      </h2>
+      <p class="text-justify">
+        YAML-LD relies upon APIs and algorithms of JSON-LD, but applies them
+        to YAML instead of JSON. Since YAML is a superset of JSON, this, in
+        most aspects, is straightforward.
```

Comment:
```suggestion
        to YAML instead of JSON. Since YAML is a superset of JSON, and JSON-LD
        is a subset of JSON, this, in most aspects, is straightforward.
```